web/projects/ui/src/app/pages/init/init.service.ts

```diff
@@ -0,0 +1,48 @@
+import { inject, Injectable } from '@angular/core'
+import { ErrorToastService } from '@start9labs/shared'
+import {
+  catchError,
+  EMPTY,
+  exhaustMap,
+  filter,
+  from,
+  interval,
+  map,
+  Observable,
+  shareReplay,
+  takeWhile,
+} from 'rxjs'
+import { ApiService } from 'src/app/services/api/embassy-api.service'
+
+@Injectable({ providedIn: 'root' })
+export class InitService extends Observable<number> {
+  private readonly api = inject(ApiService)
+  private readonly errorService = inject(ErrorToastService)
+  private readonly progress$ = interval(500).pipe(
+    exhaustMap(() =>
```

Comment:
Use guid to open websocket and get ongoing progress.

Should also return phases